tests/acceptance/blocks/resultsheadercomponent.js

```diff
@@ -0,0 +1,26 @@
+import { Selector, t } from 'testcafe';
+
+/**
+ * This class models user interactions with the {@link ResultsHeaderComponent}.
+ */
+export default class ResultsHeaderComponentBlock {
+  constructor (selector) {
+    this._container = '.yxt-ResultsHeader';
+    this._selector = selector || Selector(this._container);
```

Comment:
Is `this._selector` a CSS selector or a DOM element / node?

Reply:
ah so it's a test cafe selector, which I've been thinking of as a lazy DOM element but not sure if that's 100% accurate. https://devexpress.github.io/testcafe/documentation/guides/basic-guides/select-page-elements.html I think you can pass it either a DOM element or a css selector!